### PR TITLE
Typo fix in DataTransformers.ipynb

### PR DESCRIPTION
**Pull request by data-hound**
Under Section 2, Multiple Inputs
 
```
class MultiOutputTransformer(BaseEstimator, TransformerMixin):
    ...

class MultiOutputClassifier(KerasClassifier):

    @property
    def feature_encoder(self):
        return MultiInputTransformer(...)
```

  changed to
  
```
class MultiInputTransformer(BaseEstimator, TransformerMixin):
    ...


class MultiInputClassifier(KerasClassifier):

    @property
    def feature_encoder(self):
        return MultiInputTransformer(...)
```

### CODE REVIEW

**Comment by adriangb** [comment]
Thank you for the bug report! This has been noted and will be fixed.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-repos:
```
Comment (adriangb) [inline .pre-commit-config.yaml:1]: Are the changes here intentional?

Comment (data-hound) [reply .pre-commit-config.yaml:1]: Nope. After I ran the pre-commit test, and everything was fine, I was asked to stage this file for commit as well, and so I did.

Comment (adriangb) [reply .pre-commit-config.yaml:1]: Hmm, not sure what happened there. Would you mind reverting this change then?

**Comment by adriangb** [comment]
I'm actually in the middle of a pretty large overhaul of the docs in #174, the main goal being to make it easier to edit notebooks. Would you mind waiting until I merge #174 and then we can use this change as a case study for the effectiveness of the new system?

**Comment by data-hound** [comment]
I see the #174 has been merged. I will make the changes then.

**Comment by adriangb** [comment]
Awesome, thank you! Please let me know if you encounter any issues with the new setup.

**Comment by adriangb** [comment]
Hey @data-hound it looks like the way I set up CI won't work for forks 🤦 . I think I'll be able to get it fixed, but it might take a couple of days. That is to say, don't worry about the CI failure, and sorry for wasting your time with broken CI.

**Comment by adriangb** [comment]
I was able to make the changes. Thank you very much @data-hound !

**Comment by data-hound** [comment]
> I was able to make the changes. Thank you very much @data-hound !

Mention Not! 
Seems you did the rectification for the pre-commit-config.yaml in the branch as well - Thanks for that!

**Pull request merged**